src/html_support_files/odoc.css

```diff
@@ -903,84 +939,80 @@ td.def-doc *:first-child {
 .hljs-strong,
 .hljs-emphasis,
 .hljs-quote {
-  color: #df5000;
+  color: var(--hljs-variable);
 }
 
 .hljs-keyword,
 .hljs-selector-tag {
-  color: #a71d5d;
+  color: var(--hljs-keyword);
 }
 
 .hljs-type,
 .hljs-class .hljs-title {
-  color: #458;
+  color: var(--hljs-type);
   font-weight: 500;
 }
 
 .hljs-literal,
 .hljs-symbol,
 .hljs-bullet,
 .hljs-attribute {
-  color: #0086b3;
+  color: var(--hljs-literal);
 }
 
 .hljs-section,
 .hljs-name {
-  color: #63a35c;
+  color: var(--hljs-name);
 }
 
 .hljs-tag {
-  color: #333333;
+  color: var(--hljs-tag);
 }
 
 .hljs-attr,
 .hljs-selector-id,
 .hljs-selector-class,
 .hljs-selector-attr,
 .hljs-selector-pseudo {
-  color: #795da3;
+  color: var(--hljs-attr);
 }
 
 .hljs-addition {
-  color: #55a532;
-  background-color: #eaffea;
+  color: var(--hljs-addition);
+  background-color: var(--hljs-addition-bg);
 }
 
 .hljs-deletion {
-  color: #bd2c00;
-  background-color: #ffecec;
+  color: var(--hljs-deletion);
+  background-color: var(--hljs-deletion-bg);
 }
 
 .hljs-link {
   text-decoration: underline;
 }
 
 .VAL, .TYPE, .LET, .REC, .IN, .OPEN, .NONREC, .MODULE, .METHOD, .LETOP, .INHERIT, .INCLUDE, .FUNCTOR, .EXTERNAL, .CONSTRAINT, .ASSERT, .AND, .END, .CLASS, .STRUCT, .SIG {
-  color: #859900;;
+  color: var(--hljs-keyword);;
 }
 
 .WITH, .WHILE, .WHEN, .VIRTUAL, .TRY, .TO, .THEN, .PRIVATE, .OF, .NEW, .MUTABLE, .MATCH, .LAZY, .IF, .FUNCTION, .FUN, .FOR, .EXCEPTION, .ELSE, .TO, .DOWNTO, .DO, .DONE, .BEGIN, .AS {
-  color: #cb4b16;
+  color: var(--orange-dim);
```

Comment:
There are modifications to the highlight color for render source code, which sometimes include "color variable" and sometimes various "semantic variable".

I think it would be nice to have only semantic variable, when applicable specific to render source code